docker-compose-tests.yml

```diff
@@ -32,8 +30,13 @@ services:
       TRAVIS_BRANCH: ${TRAVIS_BRANCH}
       TRAVIS_COMMIT: ${TRAVIS_COMMIT}
       TRAVIS_JOB_ID: ${TRAVIS_JOB_ID}
+    links:
+    - profiles-db:mongo
+
+  profiles-db:
+    image: "nhsuk/profiles-db:${profiles_db_docker_image_tag-latest}"
 
 volumes:
-  test-deps:
-  travis-test-deps:
+  profiles-node_modules:
```

Comment:
This should be unique to each service to avoid problems with shared versions of `node_modules`